src/nvim/eval.c

```diff
@@ -9595,13 +9595,15 @@ static void f_getcwd(typval_T *argvars, typval_T *rettv, FunPtr fptr)
       if (from) {
         break;
       }
-    case kCdScopeTab:         // FALLTHROUGH
+      // fallthrough
```

Comment:
@jamessan My comment indeed means that, but I would not mind macros in place of comments as well.